### DISCUSSION TITLE
User is too new! Please check with support.

### DISCUSSION THREAD

**Discussion by ghost** [cirruslabs/cirrus-ci-docs#1050] [Q&A]
I Created My CirrusCI account yesterday and i can't use that.
So,How Much Time it will take unlock my account

Thanks :-)

**Comment by fkorotkov** [comment]
This is a 48 hours window. We've seen a lot of crypto miners creating accounts and just flooding CIs like Cirrus. GitHub is not allowing to enforce some extra verification for users that use Cirrus CI App so instead we implemented this ugly workaround. Sorry for the inconveniences!
